api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1758,15 +1768,14 @@ type AzurePlatformSpec struct {
 	// +immutable
 	VnetID string `json:"vnetID,omitempty"`
```

Comment:
Why is it marked as `+optional`? What is the behavior if not provided?

Reply:
It's optional. There is no behavior changes if it is not provided.

Reply:
I don't think that's clear enough.

What does "there is no behavior changes if it is not provided" mean? What vnet is used then? can the hostedcluster work at all without a vnet? is the vnet part of subnetid used instead? is vnetname used instead?